beacon/message.go

```diff
@@ -14,6 +14,7 @@ import (
 type ProposalMessage struct {
 	EpochID      types.EpochID
 	NodeID       types.NodeID
+	Nonce        types.VRFPostIndex
```

Comment:
this is incorrect. we don't need Nonce here. this is not the VRF message being signed. this is just the msg sent over the wire during the beacon protocol.